library/std/src/io/mod.rs

```diff
@@ -1361,6 +1362,7 @@ impl Initializer {
 /// [`write_all`]: Write::write_all
 #[stable(feature = "rust1", since = "1.0.0")]
 #[doc(notable_trait)]
+#[rustc_diagnostic_item = "Write"]
```

Comment:
maybe IoWrite, given that there are multiple Writes?

Reply:
Sure